React-Todo-list-main/src/Components/TodoWrapper.js

```diff
@@ -13,31 +14,40 @@ import {
   TODO_EDITED,
   TODO_LAST_VIEWED,
 } from "../Constant";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSignOut, faWarning } from "@fortawesome/free-solid-svg-icons";
 
 /**
  * Main component to display the list of todos
  * And handling statsig experiment to sort the todo list
  * Getting the feature to enable or disable the delete option
  * @returns
  */
-export const TodoWrapper = () => {
+export const TodoWrapper = ({ onLogout }) => {
   /**
    * To get the feature gate value from statsig console
    */
   const { value, isLoading } = useGate(FEATURE_GATE_1);
   /**
    * To get the experiment config from statsig console
    */
-  const { config } = useExperiment(EXPERIMENT_SORTING);
+  const { config: experimentConfig } = useExperiment(EXPERIMENT_SORTING);
+
+  /**
+   * To get the dynamic config
+   */
```

Comment:
remove comments like this - the code is the source of truth

Reply:
Agree removing it